Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx

```diff
@@ -312,6 +312,93 @@ void vtkMRMLApplicationLogic::PropagateVolumeSelection(int fit)
     }
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLApplicationLogic::PropagateBackgroundVolumeSelection(int fit)
```

Comment:
@mehrtash Earlier today, I provided comment with suggestion how to address Steve remarks.